test/journeys/specs/tap/widget-space/space.js

```diff
@@ -201,8 +182,8 @@ describe(`Widget Space: Group Space: TAP`, () => {
       verifyMessageReceipt(local, remote, lorraineText);
       verifyMessageReceipt(remote, local, lorraineText);
       const clientSendEvents = getEventLog(local.browser);
-      assert.include(clientSendEvents, `messages:created`, `has a message created event`);
-      assert.include(clientSendEvents, `rooms:unread`, `has an unread message event`);
+      assert.isTrue(clientSendEvents.some((event) => event.eventName === `messages:created`), `event was not seen`);
+      assert.isTrue(clientSendEvents.some((event) => event.eventName === `rooms:unread`), `event was not seen`);
```

Comment:
Consider adding which type of event to the error message (`messages: created event was not seen`) because there's a lot of assertions here